### ISSUE TITLE
make acookiecutter recipe

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#2689]
CC @underyx

**Comment by souravsingh** [comment]
@sils1297 We have to make a cookiecutter recipe for a bear, correct?

**Comment by sils1297** [comment]
yes!

**Comment by SanketDG** [comment]
@sils1297 shouldn't this be in a new repo?

**Comment by sils1297** [comment]
yes @souravsingh just make it a new repository, I'll create one for you when you're ready and then we'll import it?

**Comment by souravsingh** [comment]
The cookiecutter recipe for coala bears is made here-https://github.com/souravsingh/cookiecutter-bear. Anyone from the coala dev teams can contribute to the recipe.

**Comment by sils** [comment]
Nice, we can build on that
